src/ReturnBookUi.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Scanner;
```

Comment:
Unsure why the review has classed the name difference between ReturnBookUi and ReturnBookUI as being two completely seperate items. But I believe this will be fine in the merge.